public/javascripts/controllers/widget.mixin.js

```
@@ -57,6 +57,14 @@ define([
         e.preventDefault();
       });
 
+      $("input[name = show_frame]").click(function () {
```

I am doing this to default false to hide test_setup and show_branding elements if show frame is false. If I don't do that we will be showing a widget demo that is not like what the div would generate